src/alias/azext_alias/alias.py

```diff
@@ -220,7 +225,7 @@ def parse_error(self):
         return not self.alias_table.sections() and self.alias_config_str
 
     @staticmethod
-    def build_collision_table(aliases, reserved_commands, levels=COLLISION_CHECK_LEVEL_DEPTH):
+    def build_collision_table(aliases, levels=COLLISION_CHECK_LEVEL_DEPTH, enable_telemetry=True):
```

Comment:
What does telemetry have to do with building the collision table?
i.e. Why would a caller want to configure `enable_telemetry`.